lib/SqlDump/SqlDumpMySQL.php

```diff
@@ -67,7 +67,10 @@ public function __construct() {
 	 * @throws SqlDumpException
 	 */
 	public function export(array $params, string $filename): void {
-		$connect = 'mysql:host=' . $params['dbhost'] . ';dbname=' . $params['dbname'];
+		$connect = sprintf('mysql:host=%s;dbname=%s', $params[ISqlDump::DB_HOST], $params[ISqlDump::DB_NAME]);
+		if(false === empty($params[ISqlDump::DB_PORT])){
```

Comment:
To fix the code style violation:

```suggestion
		if (false === empty($params[ISqlDump::DB_PORT])) {
```